pkg/metricsservice/api/metrics.pb.go

```diff
@@ -16,7 +16,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.34.2
-// 	protoc        v5.27.3
+// 	protoc        v4.25.2
```

Comment:
please don't include these changes in the pb files, not related

Reply:
DCO fixed, and changes in pb file reverted (tbh, not sure how it got there).